test/sagemaker_tests/pytorch/training/integration/sagemaker/test_dgl.py

```diff
@@ -71,7 +71,7 @@ def test_dgl_gcn_training_gpu(ecr_image, sagemaker_regions, instance_type):
         "instance_type": instance_type,
     }
 
-    if Version(image_framework_version) in SpecifierSet(">=1.12"):
+    if Version(image_framework_version) in SpecifierSet(">=1.11.0"):
```

Comment:
What changed here?

Reply:
We haven't pinned the dgl package and the latest got updated to 0.9.1, hence the change